src/frontend/src/sass/_font.scss

```diff
@@ -1,6 +1,15 @@
-/* This section describes the default fonts we use in B.C gov
- */
-@font-face {
-  font-family: myFirstFont;
-  src: url('sansation_light.woff');
+.font__weight--700 {
```

Comment:
can we avoid this type of class and assign it directly to h1,h2 or the relevant tag? If an h2 tag somehow has a font weight of a h1 tag, the UI/UX team needs to be notified that branding is breaking. 

What are your thoughts?

Reply:
Sure. `h1`s already have this font-weight. Are we saying that elements that are not bold always, will never be bold?

Reply:
yes. 
All headings should have one font weight - H1 h2 h3 h4 etc
other elements like p or label or error should inherit the p property. p can have bold or normal if need be. 

We can confirm with branding

Reply:
if others feel this is fine, please disregard my comments.  Feel free to commit.

Reply:
So far, `h1` is never used without a bold font-weight so it definitely should have a `font-weight: 700;`; however, in general, I think this class works well for adding bold to elements where needed as per the designs, though I may rename it to `text__font-weight--700`.